### ISSUE TITLE
Add composer recipe for symfony/flex

### ISSUE THREAD

**Issue by seizan8** [Setono/SyliusGiftCardPlugin#198]
This is more a request/suggestion than a real issue/bug.

It seems there is no recipe for this plugin, which is quite the shame.
It would be nice if the config files were added automatically when requiring this Plugin. Also the templates could be added automatically, which would be really neat.

https://github.com/symfony/recipes/blob/master/README.rst

**Comment by loevgaard** [comment]
Totally agree. I have never made a recipe before. If I make the repository, are you up for the task? :)

**Comment by seizan8** [comment]
Do you need a repository for it? Maybe I misunderstood it. I thought you can simply create a PR on the contributors-repository?
https://github.com/symfony/recipes-contrib

I didn't make any recipes yet either. I am intrigued to learn it though, it doesn't seem that complicated.
However, I don't want to make any promises. If I find some spare time I would like to look into it. 
You will definitely hear from me again if I do play around with these recipes :)

**Comment by loevgaard** [comment]
Yeah, that's probably right, that you make a PR on that repository :)
